communication/requesting-survey.md

```diff
@@ -0,0 +1,102 @@
+
+If the survey you want to do is a community survey, then then next thing is to submit a request.
+
+## Survey Request Process
+
+### 1. Determine Goals and Content of Survey
+
+The first step needs to happen in your community Group.  You need to determine:
+
+1. Why you want to do a survey
+2. Who you would like to answer it
+3. What specific questions you want to ask
+
+Ideally, answer those questions in that order; first decide what your goals are, next which audiences you're targeting, and finally compose the individual questions.
+
+Your survey will also need a preface which explains to the audience what the survey is for, who will get to see its raw results, and where compiled data from it will be published.
+
+### 2. Request Survey Review
+
+Put your draft survey questions in a Google Doc, HackMD page, PR against one of your Group repos, or other format that supports comments by reviewers.  Then open an [issue](https://github.com/kubernetes/community/issues) against the community repo using this template:
+
+```
+Group Requesting:
+
+Title of Survey:
+
+Goal of Survey:
+
+Target Audience:
+
+Results Recipients:
+
+Open/Close Dates:
+
+Communications Help Wanted?  Y / N
+
+Data Analysis Wanted?  Y / N
+
+Link to Draft Questions:
+
+/sig contributor-experience
+/area survey
+```
+
+Members of SIG-ContribEx will then offer feedback on improving your survey, in the following areas:
+
+* Are the questions and options clear and unambiguous?
+* Are there obvious missing options to some questions?
+* Is the preface clear and complete?
+* Does the survey comply with the Kubernetes CoC and privacy practices?
+
+Once you feel like the survey is as good as it's likely to be, we can publish it.
+
+### 3. Request Survey Publication
+
+By default, we use a SurveyMonkey account funded by the CNCF to publish surveys.  This is because SurveyMonkey offers a fairly rich set of survey options, and more importantly is accessible in China.  We can also publish surveys using the Kubernetes Google Drive account, if surveying Chinese developers or contributors is not a consideration.  GDrive makes sharing the survey results easier.
+
+In either case, it will take a few days for a ContribEx volunteer to copy the questions and options into the survey tool, and then it can be ready to announce on your schedule.  Just comment on the issue giving your requested publication schedule, and mark it LGTM.
+
+### 4. Contributor Communications Promotion (optional)
+
+If you want your survey to reach a large number of contributors, developers, or users, then the Contributor Marketing subproject can help.  Tag `/area contributor-comms` on the issue, and request help from that group.  They will ask you some questions about promoting the survey and assist you in getting the word out, via the Kubernetes social medial accounts, mailing lists, and blog, depending on the target audience and scope.
```

Comment:
This is overall great. If it's sensible to you too @jberkus, I'd prefer a "one issue, one request" philosophy, so offering alternative text.

```suggestion
If you want your survey to reach a large number of contributors, developers, or users, then the Contributor Marketing subproject can help.  Use the [contributor comms issue template](https://github.com/kubernetes/community/issues/new/choose) to request help from that group.  They will ask you some questions about promoting the survey and assist you in getting the word out, via the Kubernetes social media accounts, mailing lists, and blog, depending on the target audience and scope.
```